pyproject.toml

```diff
@@ -11,20 +11,20 @@ packages = [{ include = "mex" }]
 [tool.poetry.dependencies]
 python = "^3.11"
 fastapi = "^0.104.1"
-httpx = "^0.25.1"
+httpx = "^0.25.2"
 mex-common = { git = "https://github.com/robert-koch-institut/mex-common.git", rev = "0.14.1" }
```

Comment:
why not update mex-common here?

Reply:
because the backend is still running on pydantic v1 and has to use the last mex-common version that does so too. the upgrade to v2 will come with mx-1485